### PR TITLE
Upgrade to 3.2.0 and other improvements

### PR DESCRIPTION
**Pull request by dfulginiti**
I pulled this from https://github.com/roundhouse/omnipay-cardconnect/pull/1 and tweaked it

### CODE REVIEW

File: src/Message/InquireRequest.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+class InquireRequest extends AbstractRequest
+{
+    public function getData()
+    {
+        $this->validate('transactionReference');
+    	$data = array(
+    		'merchid'   => $this->getMerchantId(),
+    		'retref'   => $this->getTransactionReference()
+    	);
```
Comment (amayer-hc) [inline src/Message/InquireRequest.php]: If we are replacing `array()` with `[]` we should also do it here.
```suggestion
    	$data = [
    	    'merchid' => $this->getMerchantId(),
            'retref' => $this->getTransactionReference()
    	];
```

File: src/Message/InquireRequest.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        ]);
+
+        return $this->response;
+    }
+
+    public function getEndpoint($data)
+    {
+        return $this->getEndpointBase() . "/inquire/{$data['retref']}/{$data['merchid']}";
+    }
+}
```
Comment (amayer-hc) [inline src/Message/InquireRequest.php]: Please add a newline to the end of this file.
image

**Review by amayer-hc — CHANGES_REQUESTED** [review]
A few minor comments.

File: src/Message/InquireRequest.php (hunk 2 above)
Comment (dfulginiti) [reply src/Message/InquireRequest.php]: lol

File: src/Message/InquireRequest.php (hunk 1 above)
Comment (dfulginiti) [reply src/Message/InquireRequest.php]: copy pasta error

File: src/Message/InquireRequest.php [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        ]);
+
+        return $this->response;
+    }
+
+    public function getEndpoint($data)
+    {
+        return $this->getEndpointBase()."/inquire/{$data['retref']}/{$data['merchid']}";
+    }
+}
```
Comment (dfulginiti) [inline src/Message/InquireRequest.php:43]: this doesn't show a new line but I swear it's there

Comment (amayer-hc) [reply src/Message/InquireRequest.php:43]: You are good if it doesn't show the circle with the minus sign in it.

**Review by amayer-hc — APPROVED** [review]
LGTM :100:

**Pull request merged**